eve/main.yml

```diff
@@ -268,23 +269,112 @@ stages:
             --batch --tmp-dir /var/tmp &&
             sudo chown eve:eve /var/tmp/sosreport*
           alwaysRun: true
-      - ShellCommand:
+      - ShellCommand: &copy_sosreport
           name: Copy sosreport to correct directory structure
           command: >
             mkdir -p sosreport/sosreport/single-node &&
             cp /var/tmp/sosreport* sosreport/sosreport/single-node
           alwaysRun: true
-      - Upload:
+      - Upload: &upload_report_artifact
           name: upload artifacts
           source: sosreport
           alwaysRun: true
-      - ShellCommand:
+      - ShellCommand: &resource_destroy_wait
```

Comment:
Minor, but the name is not really good here IMO, it's a debug step and for this one maybe better to keep copy paste for the moment as we may want to active debug only for one particular step